lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -12,3 +12,6 @@ statsd
 azure-storage==0.32.0
 # PyRods not in PyPI
 python-ldap==2.4.27
+
+# Synnefo / Pithos+ object store client
+kamaki
```

Comment:
this also needs something like this:
```
def check_kamaki( self ):
        return 'pithos' in self.object_stores
```

around [here](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/__init__.py#L108)